meson.build

```diff
@@ -89,10 +87,11 @@ gio_dep = dependency('gio-2.0', version: '>= @0@'.format(glib_version_required))
 gio_unix_dep = dependency('gio-unix-2.0', version: '>= @0@'.format(glib_version_required))
 gmodule_dep = dependency('gmodule-2.0')
 gtk_dep = [dependency('gtk+-3.0', version: '>= @0@'.format(gtk_version_required)), dependency('gdk-x11-3.0')]
+gtk4_dep = dependency('gtk4')
 gee_dep = dependency('gee-0.8')
-granite_dep = dependency('granite', version: '>= 5.4.0')
+granite_dep = dependency('granite-7')
 gnome_desktop_dep = dependency('gnome-desktop-3.0')
-hdy_dep = dependency('libhandy-1')
+adw_dep = dependency('libadwaita-1')
```

Comment:
Are you actually using Adw here? I think we were only using Hdy for the window

Reply:
Ah right :)